apps/openmw/mwworld/weather.cpp

```diff
@@ -289,44 +294,54 @@ WeatherManager::WeatherManager(MWRender::RenderingManager* rendering, MWWorld::F
 
     //Weather
     Weather clear;
+    clear.mObstructsCelestialBodies = false;
```

Comment:
I can't resist a good mystery.

I've been experimenting with more values, and I THINK what Clouds Maximum Percent affects is the maximum amount of transparency that the clouds can start at. That means that you ordinarily won't see this happen, since the clouds are normally opaque, BUT you can see it quite easily during a weather transition. Try changing Rain's max % to 0.0 and then change the weather from something like Clear to Rain in MW: you should see the rain clouds immediately pop into the sky, fully opaque. It's kind of funny since the sky color hasn't finished transitioning so there's a dark grey set of clouds with a cheerful blue horizon still there.